en/src/css/custom.css

```
@@ -820,4 +825,30 @@ hr {
+
+/* ================================================================
+   ACCESSIBILITY UTILITIES
+   ================================================================ */
+.visually-hidden {
+  position: absolute !important;
+  width: 1px !important;
+  height: 1px !important;
+  padding: 0 !important;
+  margin: -1px !important;
+  overflow: hidden !important;
+  clip: rect(0, 0, 0, 0) !important;
```

Stylelint (17.7.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)